src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -1999,10 +1999,10 @@ static void M()
             var compilation = CreateCompilationWithMscorlib(text);
             var tree = compilation.SyntaxTrees.Single();
             var model = compilation.GetSemanticModel(tree);
-            var position = text.IndexOf("{", text.IndexOf("a)"));
+            var position = text.IndexOf('{', text.IndexOf("a)", StringComparison.Ordinal));
```

Comment:
Well, anything that goes thru culture-specific logic is obviously slower than ordinal comparison which just compares bytes in memory without implementing culture-specific behavior. No need for benchmarks in this case.